django/contrib/admin/templates/admin/change_list.html

```diff
@@ -75,9 +75,14 @@
         {% if cl.has_filters %}
           <div id="changelist-filter">
             <h2>{% translate 'Filter' %}</h2>
-            {% if cl.has_active_filters %}<h3 id="changelist-filter-clear">
-              <a href="{{ cl.clear_all_filters_qs }}">&#10006; {% translate "Clear all filters" %}</a>
-            </h3>{% endif %}
+            {% if cl.is_facets_optional or cl.has_active_filters %}<div id="changelist-filter-extra-actions">
+              {% if cl.is_facets_optional %}<h3 id="changelist-filter-facets">
+                {% if cl.add_facets %}<a href="{{ cl.remove_facet_link }}" class="hidelink">{% translate 'Hide counts' %}</a>{% else %}<a href="{{ cl.add_facet_link }}">{% translate 'Show counts' %}</a>{% endif %}
```

Comment:
`viewlink` disappeared:
```suggestion
                {% if cl.add_facets %}<a href="{{ cl.remove_facet_link }}" class="hidelink">{% translate 'Hide counts' %}</a>{% else %}<a href="{{ cl.add_facet_link }}" class="viewlink">{% translate 'Show counts' %}</a>{% endif %}
```